integration/api/src/main/java/com/cerner/beadledom/integration/api/HelloWorldResource.java

```diff
@@ -0,0 +1,35 @@
+package com.cerner.beadledom.integration.api;
+
+import com.cerner.beadledom.integration.api.model.HelloWorldDto;
+import com.cerner.beadledom.integration.api.model.OffsetPaginatedListHelloWorldDto;
+import com.cerner.beadledom.jaxrs.GenericResponse;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Client interface of an offset based paginated list endpoint resource.
+ *
+ * @author Nick Behrens
+ */
+@Api(value = "/hello", description = "Retrieve hello world data")
+@Path("/hello")
+public interface HelloWorldResource {
```

Comment:
Since this is only for the client should it go to the client module instead of api?

Reply:
Yes, it should be alright to go there: https://github.com/cerner/beadledom/pull/149/commits/acc0165095c733471eec959e3a986d962a3146e2